tests/PerformanceTests/Microsoft.Identity.Web.Perf.Client/appsettings.json

```diff
@@ -0,0 +1,11 @@
+{
+    "IntegrationTestServicesBaseUri": "https://msidwebintegrationtestservice.azurewebsites.net",
+    "TenantDomain": "msidwebperformancetesting.onmicrosoft.com",
+    "DurationInMinutes": "1440",
+    "UserPassword": "secret_here",
+    "TestUri": "/SecurePage/GetTokenForUserAsync",
```

Comment:
is there a way for this to run auto-magically in vsts w/the other tests or run continuously? is it possible to get the lab to implement something similar so we can use them to get the password and have it run as often as needed?

Reply:
Yea, I'm sure there's a way. Maybe Azure Functions or WebJob.